vendor/github.com/docker/go-plugins-helpers/volume/api.go

```
@@ -120,3 +134,27 @@ func (h *Handler) handle(name string, actionCall actionHandler) {
 		sdk.EncodeResponse(w, res, res.Err)
 	})
 }
+
+func (h *Handler) handleMount(name string, actionCall mountActionHandler) {
+	h.HandleFunc(name, func(w http.ResponseWriter, r *http.Request) {
+		var req MountRequest
+		if err := sdk.DecodeRequest(w, r, &req); err != nil {
+			return
+		}
+
+		res := actionCall(req)
+		sdk.EncodeResponse(w, res, res.Err)
+	})
+}
+
+func (h *Handler) handleUnmount(name string, actionCall unmountActionHandler) {
+	h.HandleFunc(name, func(w http.ResponseWriter, r *http.Request) {
+		var req UnmountRequest
+		if err := sdk.DecodeRequest(w, r, &req); err != nil {
```

Same as in handleMount. Don't swallow the error.